dev-perl/Statistics-Basic/Statistics-Basic-1.661.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DIST_AUTHOR=JETTERO
+DIST_VERSION=1.6611
+inherit perl-module
+
+DESCRIPTION="Statistics-Basic module"
```

Comment:
This value of DESCRIPTION is not conforming to standards.

DESCRIPTION should never be: "The package name + the word module".

In general, if you find a description just repeats the name of the package, you're doing something wrong.
And in perl space, needing "module" in a description is usually suspicious.

Generally instead, we borrow the text verbatim from upstream's ABSTRACTS 

https://metacpan.org/pod/distribution/Statistics-Basic/lib/Statistics/Basic.pod#NAME 

This will be fixed in my rework of this ebuild, just telling you for future understanding :)